pcm_codec_registration.src.html

```diff
@@ -0,0 +1,117 @@
+
+    Linear PCM is the [[webcodecs#audio-sample-formats|raw audio format]] used
+    in WebCodecs and does not require decoding. The motivation for registering
+    the Linear PCM codec is to enable passthrough "decoding" of PCM content as
+    an architectural simpliciation for WebCodecs applications.
+
+    The registration is not intended to include any information on whether a
+    codec format is encumbered by intellectual property claims. Implementers and
+    authors are advised to seek appropriate legal counsel in this matter if they
+    intend to implement or use a specific codec format.
+
+    This registration is non-normative.
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/w3c/webcodecs">Git Repository.</a>
+!Participate: <a href="https://github.com/w3c/webcodecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/w3c/webcodecs/commits">https://github.com/w3c/webcodecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: attribute
+        text: AudioDecoderConfig.description; url: dom-audiodecoderconfig-description
+        text: AudioDecoderConfig.sampleRate; url: dom-audiodecoderconfig-samplerate
+        text: AudioDecoderConfig.channelCount; url: dom-audiodecoderconfig-channelcount
+    type: dfn
+        for: EncodedAudioChunkType; text: key; url: dom-encodedaudiochunktype-key
+        for: EncodedAudioChunk; text: [[internal data]]; url: dom-encodedaudiochunk-internal-data-slot
+        for: EncodedAudioChunk; text: [[type]]; url: dom-encodedaudiochunk-type-slot
+        text: sample; url: sample
+        text: interleaved; url: interleaved
+        text: section on sample magnitude; url: audio-samples-magnitude
+    type: interface
+        text: EncodedAudioChunk; url: encodedaudiochunk
+        text: AudioData; url: audiodata-interface
+    type: dictionary
+        text: AudioDecoderConfig; url: dictdef-audiodecoderconfig
+        text: AudioEncoderConfig; url: dictdef-audioencoderconfig
+    type: enum
+        text: AudioSampleFormat; url: enumdef-audiosampleformat
+        text: u8; url: dom-audiosampleformat-u8
+        text: s16; url: dom-audiosampleformat-s16
+        text: s24; url: dom-audiosampleformat-s24
+        text: s32; url: dom-audiosampleformat-s32
+        text: f32; url: dom-audiosampleformat-f32
+</pre>
+
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+This codec's string begins with the prefix "pcm-", followed by a suffix that
+indicates the sample format. The complete list of strings and associated
+formats is as follows.
+
+* pcm-u8, using format {{u8}}
+* pcm-s16, using format {{s16}}
+* pcm-s24, using three [[WEBIDL#idl-octet|byte]] (24-bit) [=samples=] with
+    [=interleaved=] [[webcodecs#audio-buffer-arrangement|channel arrangement]].
```

Comment:
All of those are interleaved, what makes s24 different? The way it's packed, 3 bytes by 3 bytes ?

Reply:
No difference, all interleaved, but I'm metnioning it here just because we don't have s24, so this is part of the inline dfn for that format.